packages/at_secondary_server/lib/src/connection/outbound/outbound_client.dart

```diff
@@ -301,16 +307,19 @@ class OutboundClient {
   Future<String?> plookUp(String key) async {
     var result = await lookUp(key, handshake: false);
     lastUsed = DateTime.now();
+    logger.finer('plookup result of the $key: $result');
     return result;
   }
 
   void close() {
     if (outboundConnection != null) {
       outboundConnection!.close();
+      logger.finer('Outbound connection closed');
     }
   }
 
   bool isInValid() {
+    logger.finer('connection is invalid');
```

Comment:
You are logging that the connection is invalid (a) with no additional context (which connection?) and (b) before you've actually checked if it is invalid